### PR TITLE
add experiment started time

### PR DESCRIPTION
**Pull request by gmethvin**

### CODE REVIEW

File: shoebox/conf/evolutions/shoebox/43.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# --- !Ups
+-- MySQL
+-- ======
+-- ALTER TABLE normalized_uri MODIFY url varchar(3072) NOT NULL;
+-- ALTER TABLE url MODIFY url varchar(3072) NOT NULL;
```
Comment (eishay) [inline shoebox/conf/evolutions/shoebox/43.sql]: why the `ALTER TABLE url MODIFY url varchar(3072) NOT NULL;` ?

Comment (gmethvin) [inline shoebox/conf/evolutions/shoebox/43.sql]: oh, my bad, i copied this from another evolution; will fix.

File: shoebox/conf/application.conf [hunk 2]
```diff
@@ -59,7 +59,7 @@ amazon.s3 = {
 # Mongo Resources
 # ~~~~~
 mongo = {
-#  events.server = "localhost"
+  events.server = "localhost"
```
Comment (2is10) [inline shoebox/conf/application.conf]: revert this?

**Comment by 2is10** [comment]
looks okay, merge at will

**Pull request merged**